lib/streamlit/cursor.py

```diff
@@ -11,16 +11,44 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-
-from typing import Optional, Tuple, Any
+from enum import Enum
+from typing import Optional, Tuple, Any, List
 
 from streamlit.report_thread import get_report_ctx
 
-# A "CursorPath" is a variable-length tuple of ints.
-CursorPath = Tuple[int, ...]
 
+class Container(Enum):
```

Comment:
Hm, wondering if there's a more precise name for this (may be especially confusing for developers, since we have st.container which is related but not identical to this concept). TopLevelContainer? RootContainer? Pane? Section? (See [alternatives for container](https://www.notion.so/streamlit/Expandable-What-is-the-final-name-for-this-149ccf1c81484741832e4140b98bb2ab#be72889122994da4804c0caa1cb2125d) from Horiz Layouts)

Aside: It's possible the correct name for "DeltaGenerator" is also "Container"

Reply:
Agree, and I like `RootContainer` - let's go with that! Easy to change down the road if people disagree.